playbook/appherd/roles/apache/templates/httpd.conf.j2

```diff
@@ -0,0 +1,355 @@
+#
```

Comment:
Save for the modifications to logging format, is this identical to the configuration that is currently in use on our app servers?